### PR TITLE
feat_: status-backend standard requests

### PR DESCRIPTION
**Pull request by igor-sirotin**
Just some common `status-backend` requests. 
No specific structure so far, we can refactor it when we understand how we use it.

### CODE REVIEW

File: cmd/status-backend/requests/http-client.env.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "default": {
+    "address": "localhost:12345",
+    "rootDataDir": "/Users/igorsirotin/Repositories/Status/_accounts/temp/1"
```
Comment (osmaczko) [inline cmd/status-backend/requests/http-client.env.json]: Hey `igorsirotin` :wave:

File: cmd/status-backend/requests/AcceptContactRequest.http [hunk 2]
```diff
@@ -0,0 +1,13 @@
+### Acceot Contact Request
+# @name AceptContactRequest
+POST {{address}}/statusgo/CallRPC
+
+{
+    "jsonrpc": "2.0",
+    "method": "wakuext_acceptContactRequest",
+    "params": [
+        {
+            "id": "0x0495811d8ac62e240e9ae7f05fa097938032066ff65764d97dc93a73d83262d884a75696520543899b047005f6fb91af42dcb04f635be760313cc4d26fa27f1ff7"
```
Comment (osmaczko) [inline cmd/status-backend/requests/AcceptContactRequest.http:10]: Shouldn't it be parameterized as well?

File: cmd/status-backend/requests/AcceptContactRequest.http [hunk 3]
```diff
@@ -0,0 +1,13 @@
+### Acceot Contact Request
+# @name AceptContactRequest
```
Comment (osmaczko) [inline cmd/status-backend/requests/AcceptContactRequest.http]: Acceot Acept :beers:

**Review by osmaczko — APPROVED** [review]
:partying_face:

File: cmd/status-backend/requests/AcceptContactRequest.http (hunk 2 above)
Comment (igor-sirotin) [reply cmd/status-backend/requests/AcceptContactRequest.http:10]: It should, but I was unsure of how we're going to use it, so decided to leave as is for now.
Like first see where the people go and then make concrete sidewalks on top of footpaths

**Review by dlipicar — APPROVED** [review]

**Pull request merged**